raiden/raidenwebui/src/app/components/token-network/token-network.component.ts

```diff
@@ -44,17 +53,30 @@ export class TokenNetworkComponent implements OnInit {
             });
             return;
         }
-        for (const channel of this.channelsToken) {
-            if (this.selectedToken.address === channel.token_address) {
-                this.sharedService.msg({
-                    severity: 'warn',
-                    summary: 'Warining message',
-                    detail: 'You already participate in this token network'
-                });
-                return;
-            }
-        }
-        this.displayDialog = true;
+        this.displayJoinDialog = true;
```

Comment:
I'd recommend a `this.funds.reset()` before this line too, to asure input field is empty on dialog show.